src/components/table/table-row/__snapshots__/table-row.spec.js.snap

```diff
@@ -49,7 +49,9 @@ exports[`TableRow render if is not classic theme renders a row to match the snap
 }
 
 .c2 .c14 {
-  cursor: move;
+  cursor: -webkit-grab;
+  cursor: -moz-grab;
+  cursor: grab;
```

Comment:
I made this identical change in 4bf6496 in [`FE-1569-react-dnd-and-dls-table-row`](https://github.com/Sage/carbon/commits/FE-1569-react-dnd-and-dls-table-row). Since the change is identical, it should merge cleanly without conflict.